src/components/application_manager/include/application_manager/resumption/resume_ctrl.h

```diff
@@ -124,6 +124,40 @@ class ResumeCtrl {
    */
   virtual void OnAwake() = 0;
 
+  /**
+   * @brief Returns Low Voltage signal timestamp
+   * @return Low Voltage event timestamp if event LOV VOLTAGE event occures
+   * otherwise 0
+   */
+  virtual time_t LowVoltageTime() const = 0;
+
+  /**
+   * @brief Returns Wake Up signal timestamp
+   * @return Wake Up timestamp if Wake Up signal occures
+   * otherwise 0
+   */
+  virtual time_t WakeUpTime() const = 0;
+
+  /**
+   * @brief Saves Low Voltage signal timestamp
+   */
+  virtual void SaveLowVoltageTime() = 0;
+
+  /**
+   * @brief Saves Wake Up signal timestamp
+   */
+  virtual void SaveWakeUpTime() = 0;
+
+  /**
+   * @brief Resets Low Voltage signal timestamp to zero
+   */
+  virtual void ResetLowVoltageTime() = 0;
```

Comment:
@AByzhynar check if this functions should be public in interface

Reply:
@LuxoftAKutsan Done. Sorry, commit squashed to old one.